### PR TITLE
Jww user story 18

### PR DESCRIPTION
**Pull request by iEv0lv3**
## User Story 18 | Favorites Index Lists Pets with Applications

* Update tests to reflect additional div class names
* Add spec for testing if pets with applications are listed on the favorites index page
* Update views to display pets with applications
* Add logic to favorites and applications controller to use pet application data
* Test coverage is 100%
* Functionality is tested for all edge cases I've thought of thus far

### CODE REVIEW

File: app/controllers/favorites_controller.rb [hunk 1]
```diff
@@ -1,5 +1,12 @@
 class FavoritesController < ApplicationController
   def index
+    apps = PetApplication.all
+    @pet_apps = []
+    if !apps.nil?
+      apps.each do |app|
+        @pet_apps << Pet.find(app[:pet_id])
+      end
+    end
```
Comment (TravisBorgsmiller) [inline app/controllers/favorites_controller.rb]: Good work on this method! I think this one and the other one I wrote will eventually live in the model but this works.
